modules/videoModule/index.js

```diff
@@ -206,38 +222,35 @@ export function PbVideo(videoCore_, getConfig_, pbGlobal_, pbEvents_, videoEvent
 
   function renderWinningBid(adUnit) {
     const adUnitCode = adUnit.code;
-    const options = { adUnitCode };
 
     const videoConfig = adUnit.video;
     const divId = videoConfig.divId;
+
     const adServerConfig = getAdServerConfig(videoConfig);
-    let adUrl;
-    if (adServerConfig) {
-      adUrl = gamSubmodule.getAdTagUrl(adUnit, adServerConfig.baseAdTagUrl, adServerConfig.params);
-    }
+    const winningBid = getWinningBid(adUnitCode);
+    const vastUrl = !adServerConfig ? winningBid?.vastUrl : gamSubmodule.getAdTagUrl(
```

Comment:
It seems to me all the changes to this file could instead be:
 - call `gamSubmodule.getVast` instead of `gamSubmodule.getAdTagUrl`
 - make sure the vast XML and not the URL is used.